.golangci.yml

```diff
@@ -63,6 +63,99 @@ linters-settings:
     allow-unused: false # report any unused nolint directives
     require-explanation: false # don't require an explanation for nolint directives
     require-specific: false # don't require nolint directives to be specific about which linter is being skipped
+  revive:
+    rules:
```

Comment:
what reason of this disabled-set?

maybe to better (and stable) revert logic and configure enabled-rules-set?
or just use revive default rules (no config at all)

Reply:
changed to use revive's default rules (with one exception)